website/reference/gql-tada-cli.md

```diff
@@ -0,0 +1,242 @@
+
+One of the short-comings of the LSP plugin method is that these don't run during `tsc`
+hence we added the `check` command which will run the same diagnostics logic of our LSP
+on your project.
+
+Example usage could look like the following:
+
+::: code-group
+
+```sh [npm]
+npx gql-tada check --level warn --exit-on-warn
+```
+
+```sh [pnpm]
+pnpm gql-tada check --level warn --exit-on-warn
+```
+
+```sh [yarn]
+yarn gql-tada check --level warn --exit-on-warn
+```
+
+```sh [bun]
+bun gql-tada check --level warn --exit-on-warn
+```
+
+:::
+
+### `generate-schema`
+
+| Option               | Description                                                                                            |
+| -------------------- | ------------------------------------------------------------------------------------------------------ |
+| `schema`             | A relative location from your current working directory where we should write the schema to.           |
+| `--tsconfig,-c`      | A relative location from your current working directory where we can find your `tsconfig`.             |
+| `--output,-o`        | Specify where to output the file to. (Default: The `schema` configuration option, if it's a file path) |
+| `--header,`          | A header key-value string to use when retrieving the introspection from a URL.                         |
+
+Generating your schema can sometimes be behind auth, ... with this command you can
+generate the schema from a URL and use environment variables in your shell rather than
+having to add them in the `tsconfig.json`.
+
+Example usage could look like the following:
+
+::: code-group
+
+```sh [npm]
+npx gql-tada generate-schema http://example.com --header 'Authorization: Bearer token'
+```
+
+```sh [pnpm]
+pnpm gql-tada generate-schema http://example.com --header 'Authorization: Bearer token'
+```
+
+```sh [yarn]
+yarn gql-tada generate-schema http://example.com --header 'Authorization: Bearer token'
+```
+
+```sh [bun]
+bun gql-tada generate-schema http://example.com --header 'Authorization: Bearer token'
+```
+
+:::
+
+### `generate-output`
+
+| Option                    | Description                                                                                            |
+| ------------------------- | ------------------------------------------------------------------------------------------------------ |
+| `--disable-preprocessing` | Whether to use our less-efficient legacy format. (Default: false)                                      |
+| `--tsconfig,-c`           | A relative location from your current working directory where we can find your `tsconfig`.             |
+| `--output,-o`             | Specify where to output the file to. (Default: The `tadaOutputLocation` configuration option)          |
+
+The `generate-output` command mimics the behavior of our LSP where it will look at
+your configured `schema` and output the `graphql-env` to your configured
+`tadaOutputLocation`.
```

Comment:
```suggestion
The `gql.tada generate-output` command mimics the behavior of `@0no-co/graphqlsp`, outputting the `gql.tada` output file manually. It will load the schema from the specified `schema` configuration option and write the output file.
```